toolkit/smart-contracts/offchain/src/governed_map/mod.rs

```diff
@@ -30,6 +30,9 @@ use std::ops::Neg;
 #[cfg(test)]
 mod tests;
 
+/// Inserts an entry into the governed map.
+/// If the `key` is already set, the operation fails.
+/// If the `key` is already set to the provided `value` a transaction is not submitted and the operation succeeds.
```

Comment:
Cosmetic: change order of lines above and use else if for the second case, please.